pyo3-polars/Cargo.toml

```diff
@@ -10,15 +10,18 @@ description = "PyO3 bindings to polars"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-polars = { version = "0.32.0", default_features = false }
-polars-core = { version = "0.32.0", default_features = false }
-polars-plan = { version = "0.32.0", default_features = false, optional = true }
-polars-lazy = { version = "0.32.0", default_features = false, optional = true }
+polars = { workspace = true, default-features = false }
+polars-core = { workspace = true, default-features = false }
+polars-ffi = { workspace = true, optional = true }
+polars-plan = { workspace = true, optional = true }
+polars-lazy = { workspace = true, optional = true }
 pyo3 = "0.19.0"
```

Comment:
Should all external deps be pinned to micro version here? E.g. any chance of `pyo3` being pinned to `0.19` only?

(exact pinning probably makes sense for polars libraries themselves though...)

Reply:
As I understand it in cargo. `0.19.0` is equal to `0.19.0..0.19.n`, though this is not super clear. To exactly pin it, we should type `=0.19.0`.